File: Data Structures & Algorithms/single-number/submission-0.java
class Solution {
    public int singleNumber(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
        for (int i : nums)
        {
            map.putIfAbsent(i, 0);
            map.put(i, map.get(i) + 1);
        }

        for (int i : map.keySet())
        {
            if (map.get(i) != 2)
                return i;
        }
        return 0;
    }
}

File: Data Structures & Algorithms/longest-increasing-subsequence/submission-0.cpp
class Solution {
public:
    int lengthOfLIS(vector<int>& nums) {
        vector<int> v(nums.size(),1);
        int max = 1;
        for (int i = 0;i < nums.size();i++)
        {
            for (int j = i + 1; j < nums.size();j++)
            {
                int num = v[i];
                if (num == v[j] && nums[j] > nums[i])
                {
                    v[j]++;
                    max = std::max(max, v[j]);
                }
            }
        }
        return max;
    }
};
